pkgs/misc/emulators/pcem/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchzip, wxGTK31, coreutils, SDL2, openal, alsaLib, pkg-config
+, autoreconfHook, withNetworking ? true, withALSA ? true, ... }:
+
+let inherit (stdenv.lib) optional;
+
+in stdenv.mkDerivation rec {
+  pname = "pcem";
+  version = "17";
+
+  buildInputs = [ wxGTK31 coreutils SDL2 openal ] ++ optional withALSA alsaLib;
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+
+  src = fetchzip {
+    url = "https://pcem-emulator.co.uk/files/PCemV${version}Linux.tar.gz";
+    stripRoot = false;
+    sha256 = "067pbnc15h6a4pnnym82klr1w8qwfm6p0pkx93gx06wvwqsxvbdv";
+  };
+
+  configureFlags = [ "--enable-release-build" ]
+    ++ optional withNetworking "--enable-networking"
+    ++ optional withALSA "--enable-alsa";
```

Comment:
```suggestion
    ++ lib.optionals withALSA [ "--enable-alsa" ];
```

Reply:
Since I'm only adding one element to the list, I prefer `optional` over `optionals`. I see it's widely used, and I don't see any documentation that says not to use it.